plugins/modules/eos_user.py

```diff
@@ -233,6 +234,19 @@ def validate_privilege(value, module):
         )
 
 
+def get_os_version(module):
+    os_version = "4.20.10"
+    response = run_commands(
+        module, 'show version | grep "Software image version"'
+    )
+    version_match = re.search(
+        r"Software image version:\s+([\d\.]+)", response[0], re.M
+    )
+    if version_match:
+        os_version = version_match.group(1)
```

Comment:
If we're going to compare this value with >, we should probably split this into a tuple of ints instead of hoping the string comparison will always hold. Something like this should work, based on the regex
```suggestion
        os_version = tuple(int(digit) for digit in version_match.group(1))
```